### PR TITLE
test_0x40000001: update test script of cpuid

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 2019338

Source cpuid file will compile failed because of
gcc version, use the executable file instead of
source file.

Signed-off-by: nanliu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
@PaulYuuu  Please help to review this, thanks.

(1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_0x40000001.q35: PASS (74.46 s)

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 1]
```diff
@@ -151,7 +151,7 @@
                     type = x86_cpu_test_0x40000001
                     start_vm = yes
                     test_dir = '/home'
-                    source_file = 'cpuid.src.tar.gz'
+                    source_file = 'cpuid.x86_64.tar.gz'
```
Comment (PaulYuuu) [inline qemu/tests/cfg/x86_cpu_flags.cfg:154]: It's a single binary file, I don't think we need to compress it.

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg]: > It's a single binary file, I don't think we need to compress it.

As we talked offline I updated and thanks.

**Review by PaulYuuu — APPROVED** [review]
Can you provide both results for RHEL7, RHEL8, and RHEL9?

**Comment by PaulYuuu** [comment]
> > Can you provide both results for RHEL7, RHEL8, and RHEL9?
> 
> Do you mean host or guest?

The test is for guests, so of course guest.

**Comment by nanliu-r** [comment]
@PaulYuuu  Test results PASS, request your approval, thanks.


 (1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_0x40000001.q35: PASS (57.85 s)
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_0x40000001.q35: PASS (51.48 s)

**Pull request merged**